Doc/library/asyncio-runner.rst

```diff
@@ -119,3 +119,30 @@ Runner context manager
 
       Embedded *loop* and *context* are created at the :keyword:`with` body entering
       or the first call of :meth:`run` or :meth:`get_loop`.
+
+
+Handling Keyboard Interruption
+==============================
+
+.. versionadded:: 3.11
+
+When a user hits the interrupt key (normally :kbd:`Ctrl-C`), :exc:`KeyboardInterrupt`
+exception is raised in the main thread by default.
```

Comment:
```suggestion
exception is raised in the main thread by default. However this doesn't work with :mod:`asyncio` 
because it can interrupt asyncio internals and can hang the program from exiting.

To mitigate this issue, :mod:`asyncio` handles :const:`signal.SIGINT` as follows:
```